.gitignore

```diff
@@ -4,3 +4,4 @@ composer.phar
 phpunit.xml
 php-cs-fixer.phar
 .puli/
+.idea/
```

Comment:
Is this really necessary ?

Reply:
It's just a habit from using PhpStorm, which automatically creates the `.idea` folder in the project root. I can remove it if you wish.

Reply:
I would prefer to update only the relevant files :)